apps/frontend/utils/aggregate/countAgg.ts

```diff
@@ -0,0 +1,10 @@
+import { compact, uniq } from 'lodash';
+
+export const countAgg = (
```

Comment:
Let's add a bit of docstring for these functions, countAgg, countCategoriesAgg, ...